apps/mobile-wallet/src/screens/LandingScreen.tsx

```diff
@@ -17,29 +17,27 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { useFocusEffect } from '@react-navigation/native'
-import { StackScreenProps } from '@react-navigation/stack'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
 import { useCallback, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { StatusBar } from 'react-native'
-import Animated from 'react-native-reanimated'
-import styled, { ThemeProvider, useTheme } from 'styled-components/native'
+import styled, { useTheme } from 'styled-components/native'
 
+import AnimatedBackground from '~/components/AnimatedBackground'
 import BlurredCard from '~/components/BlurredCard'
+import BottomButtons from '~/components/buttons/BottomButtons'
 import Button from '~/components/buttons/Button'
-import { ScreenProps } from '~/components/layout/Screen'
+import Screen, { ScreenProps, ScreenSection } from '~/components/layout/Screen'
 import ScreenTitle from '~/components/layout/ScreenTitle'
-import ScrollScreen from '~/components/layout/ScrollScreen'
 import { useAppDispatch, useAppSelector } from '~/hooks/redux'
 import AlephiumLogo from '~/images/logos/AlephiumLogo'
 import RootStackParamList from '~/navigation/rootStackRoutes'
 import { storedWalletExists } from '~/persistent-storage/wallet'
-import AnimatedCirclesBackground from '~/screens/Dashboard/AnimatedCirclesBackground'
 import { methodSelected, WalletGenerationMethod } from '~/store/walletGenerationSlice'
 import { DEFAULT_MARGIN } from '~/style/globalStyle'
-import { themes } from '~/style/themes'
 import { resetNavigation } from '~/utils/navigation'
 
-interface LandingScreenProps extends StackScreenProps<RootStackParamList, 'LandingScreen'>, ScreenProps {}
+interface LandingScreenProps extends NativeStackScreenProps<RootStackParamList, 'LandingScreen'>, ScreenProps {}
```

Comment:
This is how the screen looks the moment I launched the app.

1. The font color is black?
2. There is top and bottom padding within the rounded box.

<img width="862" alt="image" src="https://github.com/user-attachments/assets/0d9453fb-a9f7-465c-9196-ab85a3bf87af">

Reply:
Fixed in follow up PR!